### PR TITLE
Add concurrency to request handling

### PR DESCRIPTION
**Pull request by WojSad**
## Summary
Paralelized DB response handling
Added tests

## Test Plan

Unit and Integration tests

### CODE REVIEW

File: configuration_controller/tests/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
-coverage~=5.5
```
Comment (xbend) [inline configuration_controller/tests/requirements.txt:1]: Maybe try to maintain alphabetical order of python packages in requirements file?

File: protocol_controller/config.py [hunk 2]
```diff
@@ -6,11 +6,18 @@ class Config:
     TESTING = False
     LOG_LEVEL = os.environ.get('LOG_LEVEL', 'DEBUG')
     API_PREFIX = os.environ.get('API_PREFIX', '/sas/v1')
+    RC_RESPONSE_WAIT_TIMEOUT = int(os.environ.get('RC_RESPONSE_WAIT_TIMEOUT', 5))  # TODO Set 5 for testing only
```
Comment (xbend) [inline protocol_controller/config.py:9]: Change the default timeout to 60.
5 is too short as request processing happens every 10sec currently.

**Review by xbend — CHANGES_REQUESTED** [review]

File: protocol_controller/config.py (hunk 2 above)
Comment (WojSad) [reply protocol_controller/config.py]: done

File: configuration_controller/tests/requirements.txt (hunk 1 above)
Comment (WojSad) [reply configuration_controller/tests/requirements.txt]: done

**Review by xbend — APPROVED** [review]
LGTM

**Pull request merged**